control_toolbox/include/control_toolbox/pid.hpp

```diff
@@ -307,21 +411,22 @@ class Pid
     {
       std::cout << "Gains: p: " << p_gain_ << ", i: " << i_gain_ << ", d: " << d_gain_
                 << ", i_max: " << i_max_ << ", i_min: " << i_min_ << ", u_max: " << u_max_
-                << ", u_min: " << u_min_ << ", trk_tc: " << trk_tc_
-                << ", antiwindup: " << antiwindup_
+                << ", u_min: " << u_min_ << ", antiwindup: " << antiwindup_
                 << ", antiwindup_strat: " << antiwindup_strat_.to_string() << std::endl;
     }
 
     double p_gain_ = 0.0; /**< Proportional gain. */
     double i_gain_ = 0.0; /**< Integral gain. */
     double d_gain_ = 0.0; /**< Derivative gain. */
-    double i_max_ = 0.0;  /**< Maximum allowable integral term. */
-    double i_min_ = 0.0;  /**< Minimum allowable integral term. */
+    [[deprecated("Use antiwindup_strat_.i_max instead.")]]
+    double i_max_ = 0.0; /**< Maximum allowable integral term. */
+    [[deprecated("Use antiwindup_strat_.i_min instead.")]]
+    double i_min_ = 0.0; /**< Minimum allowable integral term. */
     double u_max_ = std::numeric_limits<double>::infinity();  /**< Maximum allowable output. */
     double u_min_ = -std::numeric_limits<double>::infinity(); /**< Minimum allowable output. */
-    double trk_tc_ = 0.0;                                     /**< Tracking time constant. */
-    bool antiwindup_ = false;                                 /**< Anti-windup. */
-    AntiwindupStrategy antiwindup_strat_ = AntiwindupStrategy::NONE; /**< Anti-windup strategy. */
+    [[deprecated("Use antiwindup_strat_ instead.")]]
+    bool antiwindup_ = false;             /**< Anti-windup. */
```

Comment:
```suggestion
    [[deprecated("Use antiwindup_strat_ instead.")]]
    bool antiwindup_ = false;             /**< Anti-windup. */
```
```suggestion
    bool antiwindup_ = false;             /**< Anti-windup. */
```

we get tons of deprecation warnings from our library itself